libwallet/assets_manager.go

```diff
@@ -196,9 +194,9 @@ func NewAssetsManager(rootDir, logDir string, netType utils.NetworkType, dexTest
 	}
 
 	// Attempt to set the log levels if a valid db interface was found.
-	if mgr.IsAssetManagerDB() {
-		mgr.GetLogLevels()
-	}
+	// if mgr.IsAssetManagerDB() {
+	// 	mgr.GetLogLevels()
+	// }
```

Comment:
why is this commented?

Reply:
oh, it should remove, it is a redundant piece of code